css/selectors/state.json

```diff
@@ -8,7 +8,7 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/semantics-other.html#selector-custom",
           "support": {
             "chrome": {
-              "version_added": false,
+              "version_added": "125",
               "notes": "From version 90 Chromium supports custom state selection using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
```

Comment:
```suggestion
              "notes": "Before version 125, custom state selection was supported using a dashed-ident (such as <code>:--foo</code>) instead of <code>:state()</code>."
```